develop/tutorials/articles/02-from-liferay-6-to-liferay-7/07-upgrading-plugins-to-liferay-7/04-upgrading-portlet-plugins/07-upgrading-a-spring-mvc-portlet.markdown

```diff
@@ -99,7 +99,7 @@ information.
 @product@'s Spring Framework JARs. If you don't rename your JARs, @product@
 assumes you're using its Spring Framework JARs and excludes yours from the
 generated WAB (Web Application Bundle).
-[Portal property `module.framework.web.generator.excluded.paths`](https://docs.liferay.com/ce/portal/7.0-latest/propertiesdoc/portal.properties.html#Module%20Framework)
+Portal property [`module.framework.web.generator.excluded.paths`](https://docs.liferay.com/ce/portal/7.0-latest/propertiesdoc/portal.properties.html#Module%20Framework)
```

Comment:
Including plain text along with preformatted text makes it stand out as a link. Otherwise, it's not evident that the preformatted text is a link. I'm going to reinstate what I had previously.